.github/ISSUE_TEMPLATE/new-release.md

```diff
@@ -26,16 +26,34 @@ future releases.
   - [ ] Push the tag with `git push $VERSION`.
   - [ ] Check that new container images are published for the tag.
     - ```
-      for i in nri-resource-policy-topology-aware nri-resource-policy-balloons nri-resource-policy-template nri-config-manager nri-memory-qos nri-memtierd; do \
-          skopeo inspect --format "$i: {{.Digest}}" docker://ghcr.io/containers/nri-plugins/$i:$VERSION;
+      for i in nri-resource-policy-topology-aware nri-resource-policy-balloons nri-resource-policy-template nri-config-manager nri-memory-qos nri-memtierd nri-sgx-epc; do
+          skopeo inspect --format "$i: {{.Digest}}" docker://ghcr.io/containers/nri-plugins/$i:$VERSION
       done
+      # Notes:
+      # You can also do an image artifact verification with this repo script:
+      ./scripts/release/check-artifacts.sh --images $VERSION
       ```
   - [ ] Finalize the new *draft* release created by CI
     - [ ] Check that all artefacts (Helm charts) were uploaded
     - [ ] Write the change log to the release.
     - [ ] Get the change log OK'd by other maintainers.
   - [ ] Publish the draft as a release.
-  - [ ] Verify that the Helm repo was updated
+  - [ ] Verify that all the Helm repo was updated
+    - ```
+      rm -f helm-index
+      wget https://containers.github.io/nri-plugins/index.yaml -Ohelm-index
+      for i in nri-resource-policy-topology-aware nri-resource-policy-balloons nri-resource-policy-template nri-memory-qos nri-memtierd nri-sgx-epc; do
+          if [ $(cat helm-index | yq ".entries.$i[] | select(.version == \"$VERSION\") | length > 0") != "true" ]; then
+              echo "FAILED: Helm chart $i:$VERSION NOT FOUND"
+          else
+              echo "OK: $i"
+          fi
+      done
+      rm helm-index
+      # Notes:
+      # You can also do an image+helm chart artifact verification with this repo script:
```

Comment:
I would remove these lines and instead just recommend using the script. But If you want to keep them, then fine.

Reply:
Why I am not eager to do that is because, at least to me, that would sound like an implicit promise that 1) this works already, will always work and 2) (and this is the part I have a problem with), it could already be used in no-brainer/cruise-control mode to decide that the release went just fine, without really thinking about it. And I'm not ready to do that yet, with such a short notice.

Reply:
Actually, I think you are right and it would make sense to remove the scriptlets and just reference the script. I'll update accordingly.